test/command_line/test_reindex.py

```diff
@@ -238,3 +239,19 @@ def test_reindex_experiments():
         )
         # Check that the scan-varying A matrices have been copied as well
         assert cryst.num_scan_points == n_scan_points
+
+
+def test_reindex_cob_op_exit(dials_data):
```

Comment:
```suggestion
def test_reindex_cb_op_exit(dials_data):
```
`cb_op` is the conventional shorthand for `change_of_basis_op` in cctbx-land